src/libs/actions/Report.ts

```diff
@@ -3644,7 +3647,9 @@ function prepareOnboardingOptimisticData(
         type: 'task',
         task: task.type,
         taskReportID: currentTask.reportID,
-        parentReportID: currentTask.parentReportID ?? '-1',
+        // API expects a string, that's why parentReportID must default to a string
+        // eslint-disable-next-line rulesdir/no-default-id-values
+        parentReportID: currentTask.parentReportID ?? '',
```

Comment:
1. Should it be possible to call this request without a valid `parentReportID`?
2. If yes - can we pass undefined instead of '' string?
3. If no - add a condition to prevent it

Reply:
@VickyStash 

I don't know that information, where can I get it?

Reply:
Ideas: 

- try to call the API call with `''` and with `undefined` as  `parentReportID` - check if it makes any difference
- check if the defaulting value always was there